example/package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "npm-dependency-submission-example",
+  "name": "@github/npm-dependency-submission-example",
```

Comment:
A hacker namespace squatted the old name and published malware to npmjs.org with it. Clever! Putting this unpublished package under `@github/` prevents future squatting.